common/src/IO/MapFileSerializer.cpp

```diff
@@ -222,7 +226,36 @@ namespace TrenchBroom {
         m_line(1),
         m_stream(stream) {}
 
-        void MapFileSerializer::doBeginFile() {}
+        void MapFileSerializer::doBeginFile(const std::vector<const Model::Node*>& rootNodes) {
```

Comment:
I think it could be useful to document, somewhere, the general control flow of the serializer because it's no longer immediately obvious. Well, it wasn't "immediately obvious" before either, but now it's even less so.

Reply:
Added some docs to the NodeSerializer class